shard.yml

```diff
@@ -0,0 +1,10 @@
+name: coala-bears
+version: 0.12.0.dev99999999999999
+
+authors:
+  - The coala developers <coala-devel@googlegroups.com>
```

Comment:
This file needs to be generated from a moban template like the others are, using Requirement classes.